traffic_portal/app/src/common/directives/treeSelect/TreeSelectDirective.d.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export namespace TreeSelectDirective {
+   export interface TreeData {
+       name: string;
+       id: string;
+       children: [TreeData];
```

Comment:
This specifies a tuple containing exactly one `TreeData`. I think it should be `TreeData[]` or `Array<TreeData>` (I tend to prefer the latter because working with Go so much gets me confused which side the `[]` goes on if I do it the reverse way in TS - but it truly does not matter which you want to use).